test/src/kbasesearchengine/test/search/ElasticIndexingStorageTest.java

```diff
@@ -282,55 +295,134 @@ public void testGenome() throws Exception {
         Assert.assertEquals("1", "" + assemblyIndex.getKeyProperties().get("contigs"));
         System.out.println("*** end testGenome***");
     }
+
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testMultiTypeSearchValidation1() throws Exception {
+
+        // null list of types
+        Set<GUID> guids = indexStorage.searchIds(null,
+                MatchFilter.create().withAccessGroupId(1),
+                null, AccessFilter.create().withAdmin(true));
+    }
+
+    @Test
+    public void testMultiTypeSearchValidation2() throws Exception {
+        indexObject("Genome", "genome01", new GUID("WS:1/1/1"), "MyGenome.1");
+
+        // empty list
+        Set<GUID> guids = indexStorage.searchIds(new ArrayList<String>(),
+                MatchFilter.create().withAccessGroupId(1),
+                null, AccessFilter.create().withAdmin(true));
+
+        Assert.assertTrue(guids.size() > 1);
```

Comment:
Most of the newer tests use the assertThat("message", got, is(expected)); form, I'd like to start moving towards that

Reply:
Done.